api/src/lib.rs

```diff
@@ -201,6 +201,22 @@ impl Client {
         self.delete::<_, SimpleApiError>(self.endpoints.source_by_id(&source_id)?)
     }
 
+    /// Delete comments by id in a source.
+    pub fn delete_comments(
+        &self,
+        source: impl Into<SourceIdentifier> + Clone,
+        comments: &Vec<CommentId>,
+    ) -> Result<()> {
+        let source_full_name = match source.clone().into() {
+            SourceIdentifier::Id(_) => self.get_source(source)?.full_name(),
```

Comment:
Nit: I think can skip the clone, which can allow you to remove it from the `source` argument bounds above:

```suggestion
        let source_full_name = match source.into() {
            source @ SourceIdentifier::Id(_) => self.get_source(source)?.full_name(),
```

Reply:
That's cool - didn't know about `@` syntax